docs/tasks.md

```diff
@@ -7,12 +7,12 @@ checkout of this repository.
 
 ## gplt::create_control_repo_from_production_env
 
-This task creates a git repo using the contents from the given node's
-puppet production environment.  Therefore, the specified node should be
+This task creates a git repo using from either a path, commonly the contents from the given node's
```

Comment:
Conflict with 'using' vs 'from'... suggested fix:
"This task creates a git repo using either a path (commonly the contents of the given node's Puppet production environment) or a public GitHub URL."